runtime/integration-tests/src/pools/token_prices_normal.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+//! Tests for token-price behaviour in a normal and healthy scenario
+//! I.e. no defaults. But we test weird tranche investment structures
+use crate::chain::centrifuge::{Amount, Call, Event, Pools, Runtime, Timestamp, Tokens, PARA_ID};
+use crate::pools::utils::*;
+use crate::pools::utils::{
+	accounts::Keyring,
+	env::{ChainState, EventRange},
+	loans::NftManager,
+	loans::{borrow_call, init_loans_for_pool, issue_default_loan, update_nav},
+	pools::{close_epoch, default_pool_calls, invest_order_call, permission_call},
+	time::secs::SECONDS_PER_DAY,
+	tokens::DECIMAL_BASE_12,
+};
+use common_types::{CurrencyId, PoolRole};
+use frame_support::traits::fungibles::Inspect;
+use fudge::primitives::Chain;
+use pallet_loans::types::Asset;
+use pallet_pools::TrancheLoc;
+use runtime_common::{AccountId, Address, Balance, InstanceId, Rate};
+use sp_arithmetic::traits::One;
+use sp_runtime::{
+	traits::AccountIdConversion, DispatchError, FixedPointNumber, Storage, TokenError,
+};
+use std::time::Duration;
+use tokio::runtime::Handle;
+
+// The block time we use for theses tests (in seconds)
+// -> two blocks per day
+const BLOCK_TIME: u64 = 86400u64;
+
+#[tokio::test]
+async fn single_tranche_investor_single_loan() {
+	// THE MANAGER MUST NOT BE DROPPED! It is the receiver of a lot of channels
+	let manager = env::task_manager(Handle::current());
+	let mut env = {
+		let mut genesis = Storage::default();
+		genesis::default_balances::<Runtime>(&mut genesis);
+		env::test_env_with_centrifuge_storage::<BLOCK_TIME>(&manager, genesis)
+	};
+
+	let mut nft_manager = NftManager::new();
+	let pool_id = 0u64;
+	let loan_amount = 10_000 * DECIMAL_BASE_12;
+	let loan_id = InstanceId(1);
+	let borrow = 9_000 * DECIMAL_BASE_12;
+	let borrow_amount = Amount::from_inner(borrow);
+	let investment = 5_000 * DECIMAL_BASE_12;
+	let maturity = time::date(90 * SECONDS_PER_DAY);
+
+	env::run!(
+		env,
+		Chain::Para(PARA_ID),
+		Call,
+		ChainState::PoolEmpty,
+		Keyring::Admin => default_pool_calls(Keyring::Admin.into(), pool_id, &mut nft_manager),
+			issue_default_loan(
```

Comment:
I missed this in the other PR, but maybe we should rename this to 
```suggestion
			issue_bullet_loan(
```

Once we have many loan types, it won't be clear what the default one is.

Reply:
Good thought! Will do.